exercises/yacht/YachtTest.cs

```diff
@@ -0,0 +1,164 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+
+public class YachtTest
+{
+    [Fact]
+    // generated but commented out/renamed as it creates collision with the 'Yacht' class name
+    // public void Yacht()
+    public void Yacht_()
```

Comment:
Ah, that is unfortunate. Modifications to the test file are a last resort, so we now have two options:

1. Rename the `Yacht` class to something else (e.g. `YachtGame`).
1. Update the test generator to generate a different name for this case.

What is your preferred solution @robkeim?

Reply:
That is unfortunate... my vote would be for the first one to keep two distinct concepts:
- The game of Yacht called `YachtGame`
- A Yacht in a game of Yacht called `Yacht`

Reply:
Agreed. I think it is just a case of setting the `Exercise` property, similar to this generator: https://github.com/exercism/csharp/blob/master/generators/Exercises/ComplexNumbers.cs#L14